unfoldingWord/translationCore
# Handling Single Chunk USFM From TS

#### This pull request addresses:
This PR address an issue were chunk files without explicit verse marker tags were being detected as missing verses, when they should be detected as the file name of the chunk. (Thats the same as the verse  number)


#### How to test this pull request:

[abu_tit_text_reg.tstudio.zip](https://github.com/unfoldingWord-dev/translationCore/files/1323094/abu_tit_text_reg.tstudio.zip)

Test this project and ensure the only missing verses are:
Titus 1:1
Titus 1:9
Titus 1:10
And all of Chapter 2


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/2728)
